README.md

```diff
@@ -27,14 +27,7 @@ Read more on [ui.nuxt.com](https://ui.nuxt.com)
 ## Installation
 
 ```bash
-# npm
-npm install @nuxt/ui
-# yarn
-yarn add @nuxt/ui
-# pnpm
-pnpm add @nuxt/ui
-# bun
-bun add @nuxt/ui
+npx nuxi@latest module add ui
 ```
 
 Then, register the module in your `nuxt.config.ts`:
```

Comment:
@danielroe I assume we can remove this section about adding it to `nuxt.config.ts`, right?

Reply:
Yes, we can. Unless you want to mention that you can configure it there.